src/modules/UI/components/PasswordReminderModal/components/PasswordInput.ui.js

```diff
@@ -0,0 +1,135 @@
+// @flow
+
+import React, { Component } from 'react'
+import { TextField } from 'react-native-material-textfield'
+
+import { THEME } from '../../../../../theme/variables/airbitz.js'
+import s from '../../../../../locales/strings.js'
+
+const DEFAULTS = {
+  secureTextEntry: true,
+  tintColor: THEME.COLORS.GRAY_2,
+  baseColor: THEME.COLORS.GRAY_2
+}
+
+const PASSWORD_TEXT = s.strings.password
+
+export type Props = {
```

Comment:
This component is a wrapper for the material textfield component that we get from a 3rd party library

This component's purpose is to customize the material textfield component with defaults that make sense in the context of entering a password, while allowing the defaults to be overridden

```
<PasswordInput />
```
 instead of 
```
<TextField 
  secureTextInput 
  tintColor={styles.tintColor} 
  baseColor={styles.baseColor} 
  label={s.strings.password} />
```

And something like this would override the defaults
```
<PasswordInput
  secureTextInput={this.state.hidePassword} 
  tintColor={this.state.error ? {...} : {...}} 
  baseColor={this.state.error ? {...} : {...}} 
  label={this.state.error ? {...} : {...}} />
```